certification/pyxis/pyxis.go

```diff
@@ -60,7 +60,7 @@ func (p *pyxisClient) createImage(ctx context.Context, certImage *CertImage) (*C
 
 	resp, err := p.Client.Do(req)
 	if err != nil {
-		log.Error(err)
+		log.Error(fmt.Errorf("%w: cannot create image in createImage", err))
```

Comment:
nit: `cannot create image in createImage` --> `cannot create image` for the log.Error log level? Seems redundant otherwise.

Reply:
Fixed.